dists/ios7/Info.plist.in

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleIcons~ipad</key>
 	<dict/>
 	<key>CFBundleIdentifier</key>
-	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<string>ScummVM</string>
```

Comment:
Why this change?
Also it is not consistent with the change made in `Info.plist`. In one you replaced the `${EXECUTABLE_NAME}` and kept the `${PRODUCT_BUNDLE_IDENTIFIER}`, and in the other you did the reverse.

Reply:
This was a too fast cut and paste.